erpnext/public/js/controllers/transaction.js

```diff
@@ -1287,6 +1287,15 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 				callback: function(r) {
 					if(!r.exc) {
 						for (let tax of r.message) {
+							// Stupid hack to avoid tax duplication
```

Comment:
wtf? There has to be a better way! Also, this should go upstream

Reply:
Yeah, I don't like it either, but there's no easy way to compare the tax template child table and the order's taxes child table.

I'll make it upstream once this is merged, in case we find a better fix.